apps/elf-council-frontend/src/ui/gsc/useGSCMembers.tsx

```diff
@@ -45,5 +45,6 @@ export function useGSCMembers(): QueryObserverResult<Delegate[]> {
     },
     queryKey: ["gsc-members"],
     enabled: !!events?.length,
+    refetchInterval: 2000,
```

Comment:
i think is a bit much, we prolly don't need to grab this all the time, its pretty static most of the time.